### PR TITLE
Implement changes from feedback #1

### PR DESCRIPTION
**Pull request by lucasiragusa**
Installed packages, replaced os.path with pathlib, added docstrings, and generated docx output. Known bugs: poor table formatting in docx and potential infinite loop for countries with many flight series.

### CODE REVIEW

File: src/landingpermit_app.egg-info/SOURCES.txt [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+src/models/flight.py
+src/models/flight_handler.py
+src/models/flight_series.py
+src/models/flight_series_handler.py
+src/models/iata_season.py
+src/models/seasons_handler.py
+src/models/ssim_file.py
+src/models/ssim_file_reader.py
+src/utils/__init__.py
+src/utils/file_helper.py
```
Comment (lucasiragusa) [inline src/landingpermit_app.egg-info/SOURCES.txt:31]: Not sure if this should be here?

**Review by lucasiragusa — COMMENTED** [review]
Made the first push, could you please review?

File: data/.DS_Store
Comment (Zaubeerer) [inline data/.DS_Store:1]: add to .gitignore and remove this file from the git repo, `git rm --cached data/.DS_Store` or similar should to the trick

File: src/landingpermit_app.egg-info/PKG-INFO [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Metadata-Version: 2.1
```
Comment (Zaubeerer) [inline src/landingpermit_app.egg-info/PKG-INFO:1]: add to .gitignore and remove it from git, as described above

File: src/lib/comparer.py [hunk 3]
```diff
@@ -1 +1,4 @@
-# TODO: what shall this be about?
\ No newline at end of file
+# TODO: what shall this be about?
+# ~The intention is that this will be used for the second part of the project, where we will 
+# Generate docs only for schedules that have changed. This file will be responsible for taking 
+# A base ssim and an alt ssim and outputing only the files that changed.
```
Comment (Zaubeerer) [inline src/lib/comparer.py:4]: Sounds good! We already have a follow-up issue, so we can merge this here as is

File: src/lib/permit_generator.py [hunk 4]
```diff
@@ -1,12 +1,13 @@
 import sys
 import os
 import pandas as pd
+from pathlib import Path
 
 # TODO: install lib as package instead
 # This will get the path to the parent directory of the current directory (i.e., landingpermit_app)
-sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+sys.path.append(str(Path(__file__).resolve().parent.parent))
```
Comment (Zaubeerer) [inline src/lib/permit_generator.py:8]: nice one!

File: src/lib/permit_generator.py [hunk 5]
```diff
@@ -15,6 +16,26 @@
 
 def generate_document(country, ssim_file, airline_name, contact_person, handler): # TODO: add type hints?
     # TODO: add docstring
+    
+    """
+    Generates a landing permit document in .docx format for a specific country.
+
```
Comment (Zaubeerer) [inline src/lib/permit_generator.py:22]: Nice docstring! 

You don't have to add the types in the docstring, as we will rather add those as type annotations and then use sphinx autodocs to automatically get the types from the type annotations.

Don't worry if this is a bit abstract now, we will cover this in the next weeks :)

File: src/models/airport.py [hunk 6]
```diff
@@ -17,3 +17,4 @@ def __init__(self, iata_code):
     def __repr__(self):
         return str(airport_data[self.iata_code])
     
+airport_data = pd.read_pickle('../data/industry/airport_data.pkl')
```
Comment (Zaubeerer) [inline src/models/airport.py:20]: better also define this path with pathlib.

You can start with cfd (current file directory) by using `pathlib.Path(__file__)` ;-)

**Review by Zaubeerer — CHANGES_REQUESTED** [review]
Hey @lucasiragusa, I have some minor suggestions to change before merging. 

Let me know if you have questions!

File: data/.DS_Store
Comment (lucasiragusa) [reply data/.DS_Store:1]: Added issue: https://github.com/lucasiragusa/landingpermit_app/issues/17

File: src/landingpermit_app.egg-info/PKG-INFO (hunk 2 above)
Comment (lucasiragusa) [reply src/landingpermit_app.egg-info/PKG-INFO:1]: Addressed in issue https://github.com/lucasiragusa/landingpermit_app/issues/17

File: src/landingpermit_app.egg-info/SOURCES.txt (hunk 1 above)
Comment (lucasiragusa) [reply src/landingpermit_app.egg-info/SOURCES.txt:31]: as per issue https://github.com/lucasiragusa/landingpermit_app/issues/17

**Review by Zaubeerer — APPROVED** [review]
looks good to me

**Pull request merged**